demos/solid_body_rotation.py

```diff
@@ -271,10 +269,6 @@ def qoi():
             for sol in sols["adjoint"][i]:
                 adj_outfile.write(sol)
 
-# In the `next demo <./solid_body_rotation_split.py.html>`__,
-# we consider solving the same problem, but splitting the solution
-# field into multiple components.
-#
```

Comment:
Please provide a link to the first Gray-Scott demo, which is now the next one in the sequence.

Reply:
Fixed in 09f4999e0ec28a5ad279975143eb80ace3601f97. I cancelled the workflow since it's just a comment.